websites/A/ADKami/presence.ts

```diff
@@ -0,0 +1,43 @@
+const presence = new Presence({ clientId: "863104116502429717" });
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  },
+
+   path: string = document.location.pathname;
+  let cat: string = path.split("/")[1];
+
+  if (cat === "drama-streaming") cat = "drama";
+
+  if (cat !== "") {
```

Comment:
Checking if it is an empty string?

Reply:
Yes, the string is empty if you are on the home page.